### PR TITLE
docs: update transaction parameters doc page

### PR DESCRIPTION
**Pull request by Torres-ssf**
- Close: https://github.com/FuelLabs/fuels-ts/issues/2469

### CODE REVIEW

**Review by maschad — DISMISSED** [review]
Nice explanations

File: apps/docs/src/guide/transactions/transaction-parameters.md [hunk 1]
```diff
@@ -2,27 +2,56 @@
...
+
+### `Max Fee`
+
+The maximum amount you're willing to pay for the transaction using the base asset. This allows users to set an upper limit on the transaction fee they are willing to pay, preventing unexpected high costs due to sudden network congestion or fee spikes.
+
+<<< @/../../docs-snippets/src/guide/transactions/transaction-parameters.test.ts#transaction-parameters-2{ts:line-numbers}
+
+### `Tip`
+
+Optional amount on the base asset to incentivise block producer to include transaction, ensuring faster processing for those willing to pay more. The value set here will be added to the transaction `maxFee`.
```
Comment (danielbate) [inline apps/docs/src/guide/transactions/transaction-parameters.md:19]:
```suggestion
An optional amount of the base asset to incentivise the block producer to include the transaction, ensuring faster processing for those willing to pay more. The value set here will be added to the transaction `maxFee`.
```

File: apps/docs/src/guide/transactions/transaction-parameters.md [hunk 2]
```diff
@@ -2,27 +2,56 @@
...
+
+### `Maturity`
+
+The number of blocks that must pass before the transaction can be included in a block. This is useful for time-sensitive transactions, such as those involving time-locked assets.
+
+For example, if the chain produces a new block every second, setting Maturity to `10` means the transaction will be processed after approximately 10 seconds.
+
+<<< @/../../docs-snippets/src/guide/transactions/transaction-parameters.test.ts#transaction-parameters-4{ts:line-numbers}
+
+### `Witness Limit`
```
Comment (danielbate) [inline apps/docs/src/guide/transactions/transaction-parameters.md:31]:
```suggestion
### Witness Limit
```

File: apps/docs/src/guide/transactions/transaction-parameters.md [hunk 3]
```diff
@@ -2,27 +2,56 @@
...
+
+The maximum byte length allowed for the transaction witnesses array. For instance, imagine a transaction that will deploy a contract. The contract bytecode will be one of the entries in the transaction witnesses. If you set this limit to `5000` and the contract bytecode length is `6000`, the transaction will be rejected because the witnesses bytes length exceeds the maximum value set.
+
+<<< @/../../docs-snippets/src/guide/transactions/transaction-parameters.test.ts#transaction-parameters-5{ts:line-numbers}
+
+### `Variable Outputs`
```
Comment (danielbate) [inline apps/docs/src/guide/transactions/transaction-parameters.md:37]:
```suggestion
### Variable Outputs
```

**Review by danielbate — DISMISSED** [review]
Great work! Nits but I felt the backticks were unnecessary given they aren't code blocks. Will approve for your consideration.

**Review by maschad — APPROVED** [review]

**Pull request merged**